wordpress-objects/Clarkson_Object.php

```diff
@@ -170,6 +170,18 @@ public function get_date( $format = 'U' ) {
 		return date( $format, strtotime( $this->_post->post_date_gmt ) );
 	}
 
+    /**
+     * Get the date localized from the wordpress .mo file.
+     *
+     * @param string $format
+     * @param bool $gmt
+     * @return string
+     */
+    public function get_localized_date( $format = 'U', $gmt = false ) {
```

Comment:
Btw, what was the reason not to use the `date_i18n()` within the template file? Because there are more date functions, and those could/should also be localized instead of just one function?